modules/ssh.go

```diff
@@ -57,6 +73,44 @@ func (s *SSHScanner) InitPerSender(senderID int) error {
 func (s *SSHScanner) GetName() string {
 	return s.config.Name
 }
-func (s *SSHScanner) Scan(t zgrab2.ScanTarget, port uint) (interface{}, error) {
-	return nil, nil
+
+func (s *SSHScanner) makeSSHGrabber(hlog *ssh.HandshakeLog) func(string) error {
```

Comment:
This probably doesn't need to return a closure and can just be inlined as part of `Scan`.

Reply:
👍 Will push change in a few